bluesky_queueserver/manager/profile_ops.py

```diff
@@ -33,10 +36,12 @@ def get_default_profile_collection_dir():
 
 _patch1 = """
 
+import sys
 import logging
+from bluesky_queueserver.manager.profile_tools import global_user_namespace
 logger_patch = logging.Logger(__name__)
 
-__local_namespace = locals()
+global_user_namespace.set_user_namespace(user_ns=locals(), use_ipython=False)
```

Comment:
do we want to start with `locals()` here or `{}`?

Reply:
I am not sure what do you mean by `{}`. The reasoning for this is that each time when we start `runpy` the references to the namespace is changing, so we need to set  the globally accessible reference to the new value (this is the reason the reference should not be stored by the objects in the plan). We do the patching only for startup files, so all the functions from external modules that are called from start up files can use the current global reference. Initially we used only `__local_namespace__` from the patched startup files.

Reply:
I mean do we want to seed this from an empty dictionary or the set of things that we have imported up until now.

Reply:
Ok, I understand the question now. We need `locals()` here.

Reply:
It is seeded from an empty dictionary before loading the first startup file.